tenant/repository.go

```diff
@@ -127,6 +128,19 @@ func (s DBService) GetNamespaces(tenantID uuid.UUID) ([]*Namespace, error) {
 
 func (s DBService) GetTenantsToUpdate(typeWithVersion map[environment.Type]string, count int, commit string, masterURL string) ([]*Tenant, error) {
 	var tenants []*Tenant
+	err := s.createGetOutdatedTenantsQuery(typeWithVersion, commit, masterURL).Limit(count).Scan(&tenants).Error
+
+	return tenants, err
+}
+
+func (s *DBService) GetNumberOfOutdatedTenants(typeWithVersion map[environment.Type]string, commit string, masterURL string) (int, error) {
+	var count int
+	err := s.createGetOutdatedTenantsQuery(typeWithVersion, commit, masterURL).Count(&count).Error
+
+	return count, err
+}
+
+func (s *DBService) createGetOutdatedTenantsQuery(typeWithVersion map[environment.Type]string, commit string, masterURL string) *gorm.DB {
```

Comment:
this func could be named `newGetOutdatedTenantsQuery` instead

Reply:
good point, renamed 37a916f